yarn.lock

```diff
@@ -7308,9 +7308,9 @@ xtend@~2.1.1:
   dependencies:
     object-keys "~0.4.0"
 
-xterm@^2.8.1:
-  version "2.9.2"
-  resolved "https://registry.yarnpkg.com/xterm/-/xterm-2.9.2.tgz#ec3e7c636ba67af4a7026be2cff7bdf08e56400a"
+xterm@theia-ide/xterm.js#v3-built:
+  version "2.9.1"
```

Comment:
strange that the yarn.lock version is 2.9.1

Reply:
It's because they forked the v3 branch while the current version was 2.9.1, and they haven't updated the version:
https://github.com/theia-ide/xterm.js/blob/v3/package.json#L4

We could change it to whatever want, but I don't think it really affects anything.